README.md

```diff
@@ -235,6 +252,57 @@ like this:
 You'll probably get a more reliable experience if you use
 [forever](https://npmjs.org/package/forever) to keep the daemon running.
 
+### Optional Arguments
+
+
+### Environment
+
+Set the env variable ```NODE_DEBUG``` to enable debugging.
+
+Example :
+
+    export NODE_DEBUG=dev,all,net,http,fs,tls,module,timers
+
+See http://www.juliengilli.com/2013/05/26/Using-Node.js-NODE_DEBUG-for-fun-and-profit/
+
+
+## Using the command line tools
+
+### pump-register-app
+
+First use this tool to create the credentials file
+
+    ./bin/pump-register-app  -t *app name*
```

Comment:
I think it's probably clearer if you write `<app name>` instead of `*app name*`. That's the standard notation I've seen used in other projects.